improver/regrid/landsea.py

```diff
@@ -313,19 +324,20 @@ def process(self, cube, input_land, output_land):
         the grids match.
 
         Args:
-            cube (iris.cube.Cube):
+            cube:
                 Cube of data to be updated (on same grid as output_land).
-            input_land (iris.cube.Cube):
+            input_land:
                 Cube of land_binary_mask data on the grid from which "cube" has
                 been reprojected (it is expected that the iris.analysis.Nearest
                 method would have been used). Land points should be set to one
                 and sea points set to zero.
                 This is used to determine where the input model data is
                 representing land and sea points.
-            output_land (iris.cube.Cube):
+            output_land:
                 Cube of land_binary_mask data on target grid.
+        Returns:
+            Cube of regridding results.
```

Comment:
You need a blank line between Args and Returns.

Reply:
Fixed